interface/wx/image.h

```diff
@@ -660,6 +663,7 @@ class wxImage : public wxObject
             @li wxBITMAP_TYPE_ICO: Load a Windows icon file (ICO).
             @li wxBITMAP_TYPE_CUR: Load a Windows cursor file (CUR).
             @li wxBITMAP_TYPE_ANI: Load a Windows animated cursor file (ANI).
+            @li wxBITMAP_TYPE_WEBP: Load a WebP file.
```

Comment:
I don't know if it should be repeated everywhere, but we probably do need to mention that WebP support is only available since 3.3.0.